service/src/main/java/bio/terra/pipelines/app/controller/TspsApiController.java

```diff
@@ -11,21 +12,21 @@
 /** Main TSPS controller */
 @Controller
 public class TspsApiController implements TspsApi {
-  private final PaoService paoService;
+  private final PipelinesService pipelinesService;
 
   @Autowired
-  public TspsApiController(PaoService paoService) {
-    this.paoService = paoService;
+  public TspsApiController(PipelinesService pipelinesService) {
+    this.pipelinesService = pipelinesService;
   }
 
-  // -- Policy Queries --
-  // TODO: PF-1733 Next step is to add group membership constraint
+  // -- Pipelines --
 
-  // -- Policy Attribute Objects --
   @Override
-  public ResponseEntity<Void> deletePao(UUID objectId) {
-    paoService.deletePao(objectId);
-    return new ResponseEntity<>(HttpStatus.NO_CONTENT);
+  public ResponseEntity<List> getPipelines() {
+    List<Pipeline> pipelineList = pipelinesService.getPipelines();
+    //    ApiPipelinesGetResult result = pipelineList;
```

Comment:
need help here - skipped this and it worked, but probably need to add something like this back in